devise-token_authenticatable.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
 
-  spec.add_dependency "devise",                         ">= 4.0.0", "< 4.2.0"
```

Comment:
Could you just include 4.2 with `">= 4.0.0", "< 4.3.0"` as a new 4.3 release might always break everything. ;-)

Could you adapt the table in the [readme](https://github.com/baschtl/devise-token_authenticatable#which-version-to-use-for-which-version-of-devise) accordingly?

Cheers!